CONTRIBUTING.md

```diff
@@ -1 +1,39 @@
 # Contributing to Real-time Ubuntu
+
+The [Real-time Ubuntu documentation] lives in the `docs/` directory and is deployed on Read the Docs.
+It is written using the [reStructuredText] format and built using [Sphinx].
```

Comment:
It is written in [reStructuredText], built with [Sphinx] , and hosted on Read the Docs.

Reply:
This is mentioned above.

Reply:
Hi @farshidtz , this edit is based on the previous suggested change which would have removed the RTD reference.

Reply:
Hi @farshidtz , please review the newly-added suggestions only. The rest of the previous comments are no longer relevant.

Thanks.